KVM/qemu/x86_cpuid.cfg

```diff
@@ -19,6 +19,10 @@
             features = "AVX_NE_CONVERT"
         - cmpccxadd:
             features = "CMPCCXADD"
+        - lam:
+            features = "lam"
```

Comment:
"lam" capital letter here if you buy in my first comment in BM/instruction-check/feature_list.py

Reply:
changed to capital.